### PR TITLE
[API] Add sylius logo to swagger

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         |  master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | part of #11250 
| License         | MIT

I added Sylius logo to swagger docs
And also I removed Webby (Webby is back :))
Screenshot 2020-04-07 at 11 49 28

### CODE REVIEW

File: templates/bundles/ApiPlatformBundle/SwaggerUi/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+{% block javascript %}
+{% if (reDocEnabled and not swaggerUiEnabled) or (reDocEnabled and 're_doc' == active_ui) %}
+    <script src="{{ asset('bundles/apiplatform/redoc/redoc.standalone.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/init-redoc-ui.js') }}"></script>
+{% else %}
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/swagger-ui-bundle.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/swagger-ui-standalone-preset.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/init-swagger-ui.js') }}"></script>
+{% endif %}
```
Comment (GSadee) [inline templates/bundles/ApiPlatformBundle/SwaggerUi/index.html.twig:88]: Wrong indentation

File: templates/bundles/ApiPlatformBundle/SwaggerUi/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+{% endblock %}
+
```
Comment (GSadee) [inline templates/bundles/ApiPlatformBundle/SwaggerUi/index.html.twig:90]: Unnecessary blank line

File: templates/bundles/ApiPlatformBundle/SwaggerUi/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        <symbol viewBox="0 0 20 20" id="large-arrow">
+            <path d="M13.25 10L6.109 2.58c-.268-.27-.268-.707 0-.979.268-.27.701-.27.969 0l7.83 7.908c.268.271.268.709 0 .979l-7.83 7.908c-.268.271-.701.27-.969 0-.268-.269-.268-.707 0-.979L13.25 10z"></path>
+        </symbol>
+
+        <symbol viewBox="0 0 20 20" id="large-arrow-down">
+            <path d="M17.418 6.109c.272-.268.709-.268.979 0s.271.701 0 .969l-7.908 7.83c-.27.268-.707.268-.979 0l-7.908-7.83c-.27-.268-.27-.701 0-.969.271-.268.709-.268.979 0L10 13.25l7.418-7.141z"></path>
+        </symbol>
+
+
```
Comment (GSadee) [inline templates/bundles/ApiPlatformBundle/SwaggerUi/index.html.twig:41]: Unnecessary blank line

File: templates/bundles/ApiPlatformBundle/SwaggerUi/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        <symbol viewBox="0 0 24 24" id="jump-to">
+            <path d="M19 7v4H5.83l3.58-3.59L8 6l-6 6 6 6 1.41-1.41L5.83 13H21V7z"></path>
+        </symbol>
+
+        <symbol viewBox="0 0 24 24" id="expand">
+            <path d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"></path>
+        </symbol>
+
```
Comment (GSadee) [inline templates/bundles/ApiPlatformBundle/SwaggerUi/index.html.twig:49]: Unnecessary blank line

**Comment by lchrusciel** [comment]
let's leave Webby :)

**Pull request closed without merging**